src/module/vue/components/sf-move-category-rows.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <article class="nogrow" :class="$style.wrapper">
+    <component
+      :is="`h${headingLevel}`"
+      :class="$style.heading"
+      :data-tooltip="$enrichMarkdown(category.dataforgedCategory?.Description)"
```

Comment:
BTW if you have custom moves, this throws an error because there's no `Description` field for that category. A good fix for that would be to alter `$enrichMarkdown` to look like this:

```ts
export function enrichMarkdown(md?: string): string {
  if (!md) return ''
  const html = marked.parse(md)
  return enrichHtml(html)
}
```